src/services/AffectService.ts

```diff
@@ -118,9 +119,13 @@ export async function putAffectCvssScore(
   delete putObject['uuid'];
   delete putObject['affect'];
   delete putObject['created_dt'];
+
+  const isV2 = osimRuntime.value?.flags?.affectsV2;
+  const apiVersion = isV2 ? 'v2beta' : 'v1';
+  const path = isV2 ? 'cvss-scores' : 'cvss_scores';
```

Comment:
:(

Reply:
Yeah... I felt the same. Turns out this change was done a while ago to adhere to http standards (using `-` instead of `_`), but we never used the `v2beta` until now, so it is unrelated to affectsV2 just happens to have the same versioning